docs/application/native/guides/text-input/autofill.md

```diff
@@ -193,13 +176,13 @@ To get the presence of Autofill data and Autofill authentication information:
         /* Error handling */
     ```
 
-4. Incase, the Autofill manager library is not needed in your application, you must destroy the Autofill manager handle using the `autofill_manager_destroy()` function:
+4. Incase, the Autofill view info handle is not needed in your application, you must destroy the Autofill view info handle using the `autofill_view_info_destroy()` function:
```

Comment:
@imfine98, please can you look at @mijin85cho suggestion "Also, can't it be merged into "Release Resources" section?"

Reply:
Moved into "Release Resource" section.